vhpc_toolkit/operations.py

```diff
@@ -2186,3 +2186,19 @@ def numa_affinity_cli(self):
                 )
 
         GetWait().wait_for_tasks(tasks, task_name="Set NUMA node affinity")
+
+    def migrate_vm_cli(self):
+        tasks = []
+        for vm_cfg in self._extract_file(self.cfg):
+            vm_obj = self.objs.get_vm(vm_cfg["vm"])
+            host_obj = self.objs.get_host(self.cfg["destination"])
+            self.logger.info(
+                f"Migrating VM {vm_cfg['vm']} to host {self.cfg['destination']}"
+            )
+            if GetVM(vm_obj).is_power_on():
+                self.logger.info(
+                    f"VM {vm_cfg['vm']} is powered on. So migration task might take some time"
```

Comment:
What's the reasoning behind this output?

Reply:
If the VM is turned on, it usually takes some time to complete the process if the VM is turned on. VCenter shows some progress bar.
Otherwise people might think the toolkit is just stuck since it doesn't print anything for a while